### DISCUSSION TITLE
Informações sobre CPF

### DISCUSSION THREAD

**Discussion by SkylineNando** [BrasilAPI/BrasilAPI#599] [General]
Olá pessoa, gostaria de saber se tem algum api que seja possível por meio do numero do CPF trazer informações do nome completo da pessoa. Alguns dias atras, entrei no sistema de emissão de nota fiscal do MEI, vi que é possível apenas com numero do CPF da pessoa localizar o nome completo dela

**Comment by narguis** [comment]
Não é possível acessar dados pessoais de uma pessoa por meio do CPF, esse tipo de proteção à privacidade é regulamentado pela LGPD (Lei Geral de Proteção de Dados). 

Ao abrir um MEI, você assume a posição de pessoa jurídica, e os dados de pessoas jurídicas são públicos, por isso seus dados ficam disponíveis para consultas públicas ao abrir um MEI. Somente organizações autorizadas, como órgãos governamentais e instituições financeiras conseguem acessar dados de pessoas físicas.

**Reply by SkylineNando** [reply]
Obrigado por compartilhar! Você sabe me dizer como poder obter essa autorização?

**Reply by narguis** [reply]
Infelimente (ou felizmente), não dá para conseguir autorização para acessar dados pessoais assim. Como eu disse, vai contra a LGPD e seria uma porta aberta para gerar casos de doxxing. Somente alguns órgãos específicos possuem esse tipo de autorização, e ainda assim estão sujeitos à regulações rígidas por parte do Governo Federal. Então, por bem ou por mal, é impossível conseguir dados pessoais através do CPF, a não ser que você seja o governo, um banco, uma seguradora, uma empresa de telefonia ou algo assim.

**Comment by SkylineNando** [comment]
compreendo, não é pra mim como pessoa física, pra uma fintech financeiro que estamos trabalho. valeu por sua atenção

**Reply by narguis** [reply]
Entendi, amigo. Nesse caso, o que posso te dizer é que você (ou seu time, seus contratantes, etc.) devem se familiarizar com os termos da LGPD, pois esse tipo de dados não pode vazar de jeito nenhum. Caso isso ocorra, vocês estão sujeitos à punição legal. 

Mesmo que seja um fintech, você ainda não vai conseguir acesso direto a algum banco de dados sensível como este. Você teria, muito provavelmente, que fechar parcerias com empresas especializadas que já tenham esse tipo de acesso, como empresas de verificação de identidade, por exemplo.

Recomendo também consultar um advogado especialista na LGPD.

Outro ponto importante, é que mesmo que você consiga fechar parcerias com uma empresa de consultas, você vai precisar de autorização explícita para cada consulta. Ou seja, ainda assim, não ia ser possível acessar qualquer CPF, você teria que deixar explícito nos seus termos que ao abrir uma conta com vocês, por exemplo, será realizada uma consulta ao CPF dessa pessoa, e aí sim, você poderia acessar os dados, mas somente dessas pessoas que buscaram seu serviço.

Então, respondendo sua pergunta inicial: Não, não existe uma API que pode buscar informações pessoais baseado no número do CPF, mas, se você tiver motivações legítimas e plausíveis para utilizar um serviço como este e seguir as regulamentações da LGPD, você pode contratar algumas empresas que realizam esse tipo de acesso (porém de uma maneira um pouco mais limitada).

Espero ter ajudado.

**Reply by narguis** [reply]
Se eu respondi sua dúvida, é uma boa ideia marcar a resposta como aceita pra tornar mais fácil que uma pessoa com essa mesma pergunta possa encontrar esse tópico no github discussions.

**Reply by cezarmezzalira** [reply]
@SkylineNando chegando de paraquedas, mas talvez com uma solução.
Existe o serviço CPFCNPJ, que é uma empresa que fornece essas informações via API, cobrando um valor por consulta.
Você compra um pacote com no minímo 500 consultas. Tem vários tipos de consulta para CPF ou CNPJ.
Segue o site da empresa: https://www.cpfcnpj.com.br/

**Reply by SkylineNando** [reply]
@cezarmezzalira muito obrigado

**Comment by kinhu** [comment]
Recomendo dar uma olhadinha nos serviços oferecidos pelo Serpro.
https://loja.serpro.gov.br/consultacpf

**Reply by SkylineNando** [reply]
Valeu mesmo irmão! muito obrigado pela sua ajuda.
